git/difffiles.go

```diff
@@ -52,15 +52,20 @@ func DiffFiles(c *Client, opt DiffFilesOptions, paths []File) ([]HashDiff, error
 			val = append(val, HashDiff{idx.PathName, idxtree, fs, uint(idx.Fsize), 0})
 			continue
 		}
-		stat, err := f.Stat()
+		stat, err := f.Lstat()
 		if err != nil {
```

Comment:
I would hope so, since it's supposed to be the same even for non-symbolic links on platforms that *do* support them.. but I just checked the os package, and Plan 9 just calls stat, while windows is slightly more complicated logic which boils down to just calling stat.